Python/testing-sdk-with-prompt-sdk/my_project/test_suites/flashcard_generator/evaluators.py

```diff
@@ -0,0 +1,134 @@
+import asyncio
+from typing import List
+
+from autoblocks.testing.models import BaseTestEvaluator
+from autoblocks.testing.models import Evaluation
+from openai import AsyncOpenAI
+
+from my_project.tasks.flashcard_generator import Flashcard
+from my_project.test_suites.flashcard_generator.test_cases import TestCase
+
+openai_client = AsyncOpenAI()
+
+
+class IsProfessionalTone(BaseTestEvaluator):
+    id = "is-professional-tone"
+
+    prompt = """Please evaluate the provided text for its professionalism in the context of formal communication.
```

Comment:
we could also use the prompt manager for these :o

Reply:
ya i was considering that too, I think i'll add it

Reply:
yeah not a blocker since it's not really what's being showcased!